app/validators/reach_root_from_department.rb

```diff
@@ -0,0 +1,22 @@
+class ReachRootFromDepartment < ActiveModel::Validator
```

Comment:
Use Department module - wrapper for this validation
module Department
  class ReachRootValidations
    ...
  end  
end